sherpa/astro/utils/smoke.py

```diff
@@ -27,10 +27,12 @@
 import sys
 import os
 
+from sherpa.utils.err import RuntimeErr
```

Comment:
I used to have my IDE set up to optimize imports automatically. I still think in those terms. Thanks for catching it.